### PR TITLE
Issue/clean up commands with env vars

### PR DESCRIPTION
**Pull request by mateuszkrasucki**
- UPD cleanup cmds with env vars + general cleanup 
    - Environment variables are now used to setup commands.
            - General config package contains common config.
            - Each commands extends upon this common config.
        - Most of the setup used for commands was moved to those
          config variables that are populated with env variables,
          secrets were omitted as they are to be fetched from vault.
        - Structured config data is populated via setting yaml file path
          as env variable and parsing it to config struct, e.g.
          components for localStatusReporter.
    - Fixes were made to handling server goruotines in commands
      after discovering unorderly exit on stopping containers.
    - Related: #20

- UPD move services config files to /services
    - Configuration files for third-party services were moved
      from root directory to /services.

### CODE REVIEW

File: cmd/batchStorageSync/main.go [hunk 1]
```diff
@@ -21,31 +20,36 @@ import (
 	"github.com/iryonetwork/wwm/utils"
 )
 
-var (
-	boltFilepath  string = "/data/batchStorageSync.db"
-	storageBucket string = "batchStorageSync"
-	storageKey    string = "lastSuccessfulRun"
+const (
+	StorageBucket string = "batchStorageSync"
+	StorageKey    string = "lastSuccessfulRun"
```
Comment (dominikznidar) [inline cmd/batchStorageSync/main.go]: why did you change the first letter to upper case?

Comment (mateuszkrasucki) [reply cmd/batchStorageSync/main.go]: I see no valid reason actually, I guess I felt like const is more important or whatever :D but since it does not need to be exported, will revert.

File: cmd/localAuth/config.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	"github.com/caarlos0/env"
+
+	"github.com/iryonetwork/wwm/config"
+)
+
+type Config struct {
+	config.Config
+	BoltDBFilepath   string `env:"BOLT_DB_FILEPATH" envDefault:"/data/localAuth.db"`
+	CloudAuthHost    string `env:"CLOUD_AUTH_HOST" envDefault:"cloudAuth"`
+	CloudAuthPath    string `env:"CLOUD_AUTH_HOST" envDefault:"auth"`
```
Comment (dominikznidar) [inline cmd/localAuth/config.go]: Environment var does not match name

File: cmd/batchStorageSync/main.go (hunk 1 above)
Comment (mateuszkrasucki) [reply cmd/batchStorageSync/main.go]: changed

File: cmd/localAuth/config.go (hunk 2 above)
Comment (mateuszkrasucki) [reply cmd/localAuth/config.go]: fixed

File: cmd/cloudAuth/main.go [hunk 3]
```diff
@@ -85,9 +90,10 @@ func main() {
 	api := operations.NewCloudAuthAPI(swaggerSpec)
 	api.ServeError = utils.ServeError
 	server := restapi.NewServer(api)
-	server.TLSPort = 443
-	server.TLSCertificate = "/certs/cloudAuth.pem"
-	server.TLSCertificateKey = "/certs/cloudAuth-key.pem"
+	server.TLSHost = cfg.ServerHost
+	server.TLSPort = cfg.ServerPort
+	server.TLSCertificate = flags.Filename(cfg.CertPath)
```
[the remark this answers is not in the document]
Comment (mateuszkrasucki) [reply cmd/cloudAuth/main.go]: this property in go-swagger generated server has this type, when you set to string literal it's ok,  you cannot use string var though.

[the remark this answers is not in the document]
Comment (dominikznidar) [reply cmd/cloudAuth/main.go]: ok

**Comment by dominikznidar** [comment]
I think we should also do something about the documentation as right now the only to discover available environment variables is to read the code. Should we perhaps list available variables together with a short description in a README inside each command?

**Comment by mateuszkrasucki** [comment]
@dominikznidar seems sensible, will also add "required" tag where needed.

**Comment by dominikznidar** [comment]
great

**Comment by mateuszkrasucki** [comment]
@dominikznidar done

File: cmd/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# IRYO WWM Services
+
+## Common configuration environment variables
+Listen below are environment variables used to populate common part of configuration for each command.
```
Comment (dominikznidar) [inline cmd/README.md]: Listed instead of listen
